modules/op-customizing-pipelines-as-code-configuration.adoc

```diff
@@ -6,7 +6,7 @@
 = Customizing {pac} configuration
 
 [role="_abstract"]
-To customize {pac}, cluster administrators can configure the following parameters in the `TektonConfig` custom resource, in the `pipelinesAsCode.settings` spec:
+To customize {pac}, cluster administrators can configure the following parameters in the `TektonConfig` custom resource, in the `platforms.openshift.pipelinesAsCode.settings` spec:
```

Comment:
should we mention `spec.` also in start?

Reply:
Not really as the word "spec" implies it.

Reply:
Example of such use: https://docs.openshift.com/pipelines/1.14/create/remote-pipelines-tasks-resolvers.html#resolver-hub-config_remote-pipelines-tasks-resolvers

Other places do have "spec." but they call it "parameter".

It's a good question of consistency to raise on a higher level, I guess. I'll try to raise it. Meanwhile can we go on with this merge?

Reply:
I have raised it with other writers and the answers I got were generally in support of the way I did it